src/openzaak/templates/admin/nlx_config.html

```diff
@@ -0,0 +1,61 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static %}
+
+{% block extrastyle %}{{ block.super }}
+    <link href="{% static 'libs/prism/prism.css'%}" rel="stylesheet" />
+
+{% endblock %}
+
+
+{% block title %} {% trans "Importeer catalogus" %} {{ block.super }} {% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; <a href="#">{% trans 'Configuratie' %}</a>
+&rsaquo; {% trans 'NLX configuratie' %}
+</div>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans 'NLX configuratie' %}</h1>
+    <div id="content-main">
+        <form action="" method="POST" enctype="multipart/form-data">
+            <div>
+                <fieldset class="module aligned ">
+
+                    <p>
+                        Gebruik onderstaand voorbeeld voor de configuratie van de NLX-inway.
+                    </p>
+                    <p>
+                        U kunt uw <code>service-config.toml</code> bestand aanpassen met de configuratie onder <code>[services]</code>.
+                    </p>
+
+                    <pre class="language-toml"><code>[services]
+{% for service in services %}
+    [services.OpenZaak-{{ service.name }}]
```

Comment:
@joeribekker what do we want to do about casing? so far I've always defined everything lower-cased

Reply:
Hmm. I wanted to see a nicer name in the NLX directory but it seems NLX doesn't care too much since I opened a [ticket about adding a pretty name variable 6 months ago](https://gitlab.com/commonground/nlx/nlx/issues/658).

Oh well, it's just for sysadmins/developers I guess, so we can use lower-case to keep it consistent.

Reply:
Service name is changed to lower-case